app/lib/scorecard/models.dart

```diff
@@ -7,7 +7,13 @@ import 'dart:io';
 import 'package:clock/clock.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:pana/models.dart'
-    show LicenseFile, PanaRuntimeInfo, Report, ReportSection, UrlProblem;
+    show
```

Comment:
At this point we may just import it unconditionally?

Reply:
Done